include/config_defaults.h

```diff
@@ -203,6 +203,12 @@
 #ifndef GAME_BRICK_BREAKER
 #define GAME_BRICK_BREAKER 0
 #endif
+#ifndef TOOL_FLASHLIGHT
+#define TOOL_FLASHLIGHT 0
+#endif
+#ifndef TOOL_CALCULATOR
+#define TOOL_CALCULATOR 0
```

Comment:
I think we should enable them by default - BUT see also my comment inside the `main.cpp`!

Reply:
Sorry, I couldt find the comment you refer too.